docs/intro/features.rst

```diff
@@ -217,26 +217,56 @@ on Automatic Spelling Correction for Russian:
 :doc:`Ranking component </components/neural_ranking>`
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Based on `LSTM-based deep learning models for non-factoid answer selection <https://arxiv.org/abs/1511.04108>`__. The
-model performs ranking of responses or contexts from some database by their relevance for the given context.
+The main neural ranking model based on `LSTM-based deep learning models for non-factoid answer selection
+<https://arxiv.org/abs/1511.04108>`__. The model performs ranking of responses or contexts from some database by their
+relevance for the given context.
+
+There are 3 alternative neural architectures available as well:
```

Comment:
Please, add 2 tables from this pull request description to the "comparison with other models" section for the ranking component in this file, add references to models configs from these 2 tables.

Reply:
done